### PR TITLE
Add `importance` groups from #14

### PR DESCRIPTION
**Pull request by addyosmani**
r @yoavweiss

### CODE REVIEW

File: EXPLAINER.md [hunk 1]
```diff
@@ -34,12 +34,11 @@ the signals.
...
-* We will define a new set of standard `fetch-groups` that will map to current
-  browser priorities: `critical`, `fonts`, `functional`, `visual`, `late`
-    * Names TBB (to be bikeshedded)
-* Developers would be able to assign resources into those groups or
+* We will define a new standard `importance` attribute that will map to current
+  browser priorities: `critical`, `high`, `medium`, `low`, `unimportant`
+* Developers would be able to assign resources into one of these importance groups or
   define resources as more or less important than said groups.
-* Developers would be able to define custom fetch groups, and assign
+* Developers would be able to define custom importance groups, and assign
```
Comment (yoavweiss) [inline EXPLAINER.md]: do we still need the custom bits?

Comment (addyosmani) [reply EXPLAINER.md]: Good question. We should drop it.

**Comment by addyosmani** [comment]
@yoavweiss Updated. PTAL when you get a chance :)

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**